src/main/java/com/three_tech_solutions/slot_app/dto/StudentResponseDTO.java

```diff
@@ -0,0 +1,9 @@
+package com.three_tech_solutions.slot_app.dto;
+
+import lombok.Data;
+
+@Data
+public class StudentResponseDTO {
```

Comment:
Mismo comentario que antes, no hace falta que especifiquemos que es un "DTO", ya con "StudentResponse" es suficiente

```suggestion
public class StudentResponse {
```